src/commands.ts

```diff
@@ -586,29 +533,8 @@ export function registerCommands(context: vscode.ExtensionContext, reposManager:
 	}));
 
 	context.subscriptions.push(vscode.commands.registerCommand('review.openFile', (value: GitFileChangeNode | vscode.Uri) => {
-		const uri = value instanceof GitFileChangeNode ? value.filePath : value;
-
-		const activeTextEditor = vscode.window.activeTextEditor;
-		const opts: vscode.TextDocumentShowOptions = {
-			preserveFocus: true,
-			viewColumn: vscode.ViewColumn.Active
-		};
-
-		// Check if active text editor has same path as other editor. we cannot compare via
-		// URI.toString() here because the schemas can be different. Instead we just go by path.
-		if (activeTextEditor && activeTextEditor.document.uri.path === uri.path) {
-			opts.selection = activeTextEditor.selection;
-		}
-
-		vscode.commands.executeCommand('vscode.open', uri, opts);
-	}));
-	context.subscriptions.push(vscode.commands.registerCommand('pr.openChangedFile', (value: GitFileChangeNode) => {
-		const openDiff = vscode.workspace.getConfiguration().get('git.openDiffOnClick');
-		if (openDiff) {
-			return vscode.commands.executeCommand('pr.openDiffView', value);
-		} else {
-			return vscode.commands.executeCommand('review.openFile', value);
-		}
+		const command = value instanceof GitFileChangeNode ? value.openFileCommand() : openFileCommand(value);
+		vscode.commands.executeCommand(command.title, command.arguments);
```

Comment:
typo here, should be `command.command`, and also `...` on arguments

Reply:
Thanks for catching!